.ci/podSpecs/distribution.yml

```diff
@@ -44,27 +55,43 @@ spec:
             -XX:+UseContainerSupport
         - name: DOCKER_HOST
           value: tcp://localhost:2375
+        - name: ZEEBE_CI_SHARED_DATA
+          value: /home/shared
       resources:
         limits:
           cpu: 2
           memory: 4Gi
         requests:
           cpu: 2
           memory: 4Gi
+      securityContext:
+        privileged: true
+      volumeMounts:
+        - name: shared-data
+          mountPath: /home/shared
+          mountPropagation: Bidirectional
     - name: golang
       image: golang:1.13.4
       command: ["cat"]
       tty: true
       resources:
         limits:
           cpu: 3
-          memory: 4Gi
+          memory: 1Gi
```

Comment:
I think we increased it because we saw some builds failing in the linker phase. Maybe we can increase it to just 2Gi to be safe?